agent/command/git_merge_pr.go

```diff
@@ -102,17 +102,10 @@ func (c *gitMergePr) Execute(ctx context.Context, comm client.Communicator, logg
 
 	githubCtx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
 	defer cancel()
-	mergeMethod := conf.ProjectRef.CommitQueue.MergeMethod
-	title := patchDoc.GithubPatchData.CommitTitle
-	if mergeMethod == githubMergeMethodMerge {
-		// if the merge method is to add a merge commit, send no title to the github API so that they use the default merge commit title
-		title = ""
-	}
 
 	mergeOpts := &github.PullRequestOptions{
-		MergeMethod: mergeMethod,
-		CommitTitle: title,
-		SHA:         patchDoc.GithubPatchData.HeadHash,
+		MergeMethod:        conf.ProjectRef.CommitQueue.MergeMethod,
+		DontDefaultIfBlank: true, // note this means that we will never merge with the default message (concatenated commit messages)
```

Comment:
This is kind of an odd variable name. Could we do DefaultIfBlank = false instead? It looks like we only use this structure in two places, so adding DefaultIfBlank = true to the other one seems feasible and less confusing.

Reply:
Oh I see. This isn't our structure 😭 that's so upsetting.

Reply:
guess who named the field in the library though 😰